src/Providers/ConsoleServiceProvider.php

```diff
@@ -101,7 +110,34 @@ class ConsoleServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        $this->commands($this->commands);
+        $this->commands($this->resolveCommands());
+    }
+
+    /**
+     * Resolve package commands
+     *
+     * @return array
+     */
+    private function resolveCommands()
+    {
+        $commands = [];
+
+        foreach (config('modules.commands', $this->commands) as $command) {
+            $commands[] = $this->resolveCommandNamespace($command);
+        }
+
+        return $commands;
+    }
+
+    /**
+     * Resolve command namespace
+     *
+     * @param string $command
+     * @return void
+     */
+    private function resolveCommandNamespace(string $command)
```

Comment:
Is this still needed when using `::class` ?

Reply:
Because `ConsoleServiceProvider` NEEDS to use `USE` for the classes for conflicting namespaces this check is needed.

Example: `Nwidart\Modules\Providers\Nwidart\Modules\Console\CommandName`

This lets the config to be cleaner.

Although, I will merge it into one method.

Reply:
Because it uses `::class` notation it conflicts